templates/individual_response/mobile_confirmation.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'layouts/_base.html' %}
```

Comment:
I don't think mobile_confirmation is the best name for this template. `sms_confirmation` or `text_message_confirmation` would be more explicit

Reply:
I've gone with `confirmation-text-message.html` and renamed the other template to `confirmation-post.html`.